web/src/hooks/useTempoResources.ts

```diff
@@ -55,7 +55,11 @@ export const useTempoResources = () => {
           setTempoResources(response.data);
           setError(undefined);
         } else {
-          throw new FetchError('', 500, response);
+          throw new FetchError(
+            'Invalid response returned from Tracing UI plugin backend service',
```

Comment:
I think this error message might be too specific and technical in its description. Something like "Error retrieving Tempo Resources" conveys the same information while being easier to understand.

See https://www.patternfly.org/ux-writing/error-messages for guidelines on error messages.